docs/themes/thxvscode/layouts/partials/break-on-capitals.html

```diff
@@ -0,0 +1,19 @@
+{{ $input := . }}
```

Comment:
It looks like the other html files in this dir are using `camelCase` rather than `snake-case` - we should probably rename this file to match the existing convention.